### DISCUSSION TITLE
Add support for LINE notification

### DISCUSSION THREAD

**Discussion by Aomaru-S** [upptime/upptime#996] [Ideas]
Hi

Thankyou for nice tool!

I suggest support for LINE notification.
LINE is the most used contact tool in JAPAN.
If LINE notifications were implemented, Japanese could more easily try out the notification feature.

I would appreciate your consideration.

*This is my first contributing to open source. I apologize for any inappropriate behavior.

**Comment by AnandChowdhary** [comment]
Good idea! Happy to accept a PR :)

**Comment by Aomaru-S** [comment]
I'm sorry. Since LINE Notify will be discontinued on March 31, 2025, this discussion will be closed.
